packages/kbn-apm-synthtrace/src/lib/apm/aggregators/service_metrics_aggregator.ts

```diff
@@ -86,10 +84,10 @@ export class ServicMetricsAggregator implements StreamAggregator<ApmFields> {
               },
             },
             failure_count: {
-              type: { type: 'long' },
+              type: 'long',
             },
             success_count: {
-              type: { type: 'long' },
+              type: 'long',
```

Comment:
Did this ever work? Nested `type` looks odd

Reply:
No it didn't work.